lib/Authorization/Exception/AccountServerException.php

```diff
@@ -0,0 +1,7 @@
+<?php declare(strict_types=1);
```

Comment:
보통 declare(strict_types=1)을 한줄 내려서 쓰는데.. 한번 확인해보심이
```suggestion
<?php
declare(strict_types=1);
```

Reply:
전역 검색해봤더니 php-oauth2 에서는 혼용해서 사용하고 있는 것 같습니다! 변경했다가 로그가 너무 복잡해질 것 같아서, 통일하려면 다른 태스크에서 하는 게 좋을 것 같습니다.